installer/debian/algorand/postinst

```diff
@@ -35,3 +35,11 @@ if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-decon
 		deb-systemd-invoke $_dh_action algorand.service >/dev/null || true
 	fi
 fi
+
+if [ "$1" = "configure" ]; then
+    if [ -f /var/lib/algorand/genesis.json-2.1.5-patch ]; then
+        # 2.1.5 bug fix - restore user-modified genesis.json
+        mv /var/lib/algorand/genesis.json-2.1.5-patch /var/lib/algorand/genesis.json
```

Comment:
maybe
```
mv -f /var/lib/algorand/genesis.json-2.1.5-patch /var/lib/algorand/genesis.json
```
to avoid files modes that won't permit writing ?

Reply:
Will fix.